crates/nu-command/src/network/http/get.rs

```diff
@@ -177,7 +177,11 @@ fn helper(
     let span = args.url.span();
     let ctrl_c = engine_state.ctrlc.clone();
     let (requested_url, _) = http_parse_url(call, span, args.url)?;
-    let redirect_mode = http_parse_redirect_mode(args.redirect)?;
+    let redirect_mode = args
+        .redirect
+        .map(RedirectMode::try_from)
+        .transpose()?
+        .unwrap_or_default();
```

Comment:
After implementing @WindSoilder 's suggestion to use the `FromStr` trait to parse `RedirectMode`, I'm not too happy with having these lines in every single one of the subcommands. The alternative I considered was implementing `TryFrom<Option<Spanned<String>>>` for `RedirectMode`, but that seems even uglier. Any ideas?

Reply:
Oh....Sorry my bad, I thought we need to convert from `&str` to `RedirectMode`, but actually we need to convert from `Span<String>` to `RedirectMode`.

After comparing these implementation, I think your original implementation is easier to understand.